### PR TITLE
typeinc: add page

### PR DESCRIPTION
**Pull request by AnirudhG07**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
Hi @AnirudhG07. Thanks for your contribution. Can you please fix the error?

> The [build](https://github.com/tldr-pages/tldr/actions/runs/9352375460) for this PR failed with the following error(s):
> 
> ```
> pages/common/typeinc.md:18: TLDR008 File should contain no trailing whitespace
> ```
> 
> Please fix the error(s) and push again.

**Comment by AnirudhG07** [comment]
I used tldr-lint, which is not showing any error, plus i manually checked, their should be no trailing spaces. :<

File: pages/common/typeinc.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# typeinc
+
+> Typeinc, an ncurses based terminal typing speed test program, written in python.
```
Comment (spageektti) [inline pages/common/typeinc.md:3]:
```suggestion
> An ncurses based terminal typing speed test program, written in Python.
```
We avoid mentioning the command name in the description.

Comment (AnirudhG07) [reply pages/common/typeinc.md:3]: Okie, anything else? Ill fix everything in 1 commit.

File: pages/common/typeinc.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+> Try out different difficulty levels and improve your typing speed.
+> More information: <https://github.com/AnirudhG07/Typeinc>.
+
```
Comment (spageektti) [inline pages/common/typeinc.md:6]: Can you please add an example for entering the typing test?
```suggestion

- Enter the typing test:
`typeinc`

```

Comment (AnirudhG07) [reply pages/common/typeinc.md:6]: Sure

File: pages/common/typeinc.md (hunk 1 above)
Comment (spageektti) [reply pages/common/typeinc.md:3]: > Okie, anything else? Ill fix everything in 1 commit.

I added one more suggestion.

File: pages/common/typeinc.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# typeinc
+
+> An ncurses based terminal typing speed test program, written in python.
```
Comment (kbdharun) [inline pages/common/typeinc.md:3]:
```suggestion
> An `ncurses` based terminal typing speed test program, written in python.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by AnirudhG07** [comment]
I have made all the changes. Thanks for the suggestions!

**Review by spageektti — APPROVED** [review]

**Pull request merged**